synapse/storage/database.py

```diff
@@ -1474,6 +1498,20 @@ async def simple_delete_many(
         keyvalues: Dict[str, Any],
         desc: str,
     ) -> int:
+        """Executes a DELETE query on the named table.
+
+        Filters rows by if value of `column` is in `iterable`.
```

Comment:
```suggestion
        Filters rows by whether the value of `column` is an `iterable`.
```

Reply:
Heh, so I copied this from below. :) I'll update both spots though.

Reply:
For the record this filters the rows by whether the value is *in* the iterable, not whether it *is* an iterable.